package.json

```diff
@@ -21,7 +21,7 @@
   "scripts": {
     "run": "esy x ODiffBin",
     "test": "esy x RunTests.exe",
-    "test-js": "esy ava",
+    "test-js": "esy ava --verbose",
```

Comment:
Added the verbose flag to better see, which tests are failing and which are passing.
Also, without this flag, ava does not report that some tests timed out.